multik-api/src/main/kotlin/org/jetbrains/kotlinx/multik/ndarray/operations/Add.kt

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright 2020-2021 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package org.jetbrains.kotlinx.multik.ndarray.operations
+
+import org.jetbrains.kotlinx.multik.ndarray.data.*
+
+public fun <T : Number> D1Array<T>.append(n: D1Array<T>): D1Array<T>{
+    val totalSize :Int = this.size+n.size
```

Comment:
coding conventions not followed. In Multik the general conventions are - https://kotlinlang.org/docs/coding-conventions.html#names-for-backing-properties.